### ISSUE TITLE
Details about finetune breathing data

### ISSUE THREAD

**Issue by ZhangzhengTomiHoYo** [3d-motion-magnification/3d-motion-mag#6]
Very great work, the success of the method is impressive. However, I still have some questions. Specifically, for the breathing data of tripod, is there any difference in the fine-tuning input during the training of 001 ~ 029 to get a model? Their inputs are all just 000 model and the same input data. They do not seem to be any different (Maybe I misunderstood it). (The difference in the lego data is obvious, as they have 30 folders)
If 001-029 is no different, how do they express tiny movements? 
I really look forward to your answering my questions

**Comment by brandonyfeng** [comment]
Thanks for your interests!
For those real-world scenes captured with tripod, the entire capture is just a monocular video, so these scenes have a different data loader than the Blender scenes like Lego. The preprocessing and training data preparation logic are different.

Let's assume the tripod video has 30 frames in total.
When we train the 000 model, we optimize this model using those 30 video frames for multi-view supervision.
When we finetune the 001-029 models, we now want each model to cover one specific timestep. In this stage, for each timestep, we only have one observed frame, and we finetune the model over this single observed viewpoint at this timestep.

In other words, for Blender scenes, we observe T timesteps, each from N viewpoint; for real-world scenes, we still observe T timesteps, but each from N=1 viewpoint only. For Blender scenes, the 001-029 models are each optimized using N images; but for real-world scenes, we can only access N=1 image to compute the loss.

**Comment by ZhangzhengTomiHoYo** [comment]
Thanks for your detailed reply and great work. Your explanation of finetune is very helpful!
